repos/fdbt-site/src/pages/editFareStageMatching.tsx

```diff
@@ -0,0 +1,404 @@
+                                handleAutoPopulateButtonClick,
+                                'top',
+                            )}
+                            <FormElementWrapper errors={errors} errorId={errorId} errorClass="govuk-radios--error">
+                                <table className="govuk-table">
+                                    <thead className="govuk-table__head">
+                                        <tr className="govuk-table__row">
+                                            <th
+                                                scope="col"
+                                                className="govuk-table__header govuk-!-width-one-half"
+                                                id="stop-name-header"
+                                            >
+                                                Stop name
+                                            </th>
+                                            <th
+                                                scope="col"
+                                                className="govuk-table__header govuk-!-width-one-quarter"
+                                                id="naptan-code-header"
+                                            >
+                                                Naptan code
+                                            </th>
+                                            <th
+                                                scope="col"
+                                                className="govuk-table__header govuk-!-width-one-quarter"
+                                                id="fare-stage-header"
+                                            >
+                                                Fare stage
+                                            </th>
+                                        </tr>
+                                    </thead>
+                                    <tbody className="govuk-table__body">
+                                        {[...stopItems].map((item) => (
+                                            <tr key={item.atcoCode} className="govuk-table__row">
+                                                <td className="govuk-table__cell stop-cell" id={`stop-${item.index}`}>
+                                                    {item.stopName}
+                                                </td>
+                                                <td
+                                                    className="govuk-table__cell naptan-cell"
+                                                    id={`naptan-${item.index}`}
+                                                >
+                                                    {item.naptanCode}
+                                                </td>
+                                                <td className="govuk-table__cell stage-cell">
+                                                    {/* eslint-disable-next-line jsx-a11y/no-onchange */}
```

Comment:
Does this work? it says its disabling the next line, but the onChange is a few lines below.

Reply:
yeah, it works and that is why it is allowing me to push